core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -64,8 +64,8 @@ class MetadataCacheControllerNodeProvider(val metadataCache: kafka.server.Metada
  * This provider is used when we are in KIP-500 mode.
  */
 class RaftControllerNodeProvider(val metaLogManager: MetaLogManager,
-                                 controllerConnectNodes: Seq[Node]) extends ControllerNodeProvider with Logging {
-  val idToNode = controllerConnectNodes.map(node => node.id() -> node).toMap
+                                 controllerQuorumVotersNodes: Seq[Node]) extends ControllerNodeProvider with Logging {
```

Comment:
perhaps just `quorumVoterNodes`

"VotersNodes" has two plurals which is confusing

Reply:
Yea. I thought about this. Figured this would trade grammar for consistency. I'm not too inclined though. So, I can change it